scripts/deployment/phase2-assets/collaterals/deploy_morphoeUSD.ts

```diff
@@ -0,0 +1,94 @@
+import fs from 'fs'
+import hre from 'hardhat'
+import { getChainId } from '../../../../common/blockchain-utils'
+import { baseL2Chains, networkConfig } from '../../../../common/configuration'
+import { fp } from '../../../../common/numbers'
+import { expect } from 'chai'
+import { CollateralStatus } from '../../../../common/constants'
+import {
+  getDeploymentFile,
+  getAssetCollDeploymentFilename,
+  IAssetCollDeployments,
+  getDeploymentFilename,
+  fileExists,
+} from '../../common'
+import {
+  eUSD_ORACLE_TIMEOUT,
+  eUSD_ORACLE_ERROR,
+  eUSD_USD_FEED,
+  PRICE_TIMEOUT,
+  DELAY_UNTIL_DEFAULT,
+} from '../../../../test/plugins/individual-collateral/meta-morpho/constants'
+import { MetaMorphoFiatCollateral } from '../../../../typechain'
+import { ContractFactory } from 'ethers'
+
+async function main() {
+  // ==== Read Configuration ====
+  const [deployer] = await hre.ethers.getSigners()
+
+  const chainId = await getChainId(hre)
+
+  console.log(`Deploying Collateral to network ${hre.network.name} (${chainId})
+    with burner account: ${deployer.address}`)
+
+  if (!networkConfig[chainId]) {
+    throw new Error(`Missing network configuration for ${hre.network.name}`)
+  }
+
+  // Get phase1 deployment
+  const phase1File = getDeploymentFilename(chainId)
+  if (!fileExists(phase1File)) {
+    throw new Error(`${phase1File} doesn't exist yet. Run phase 1`)
+  }
+  // Check previous step completed
+  const assetCollDeploymentFilename = getAssetCollDeploymentFilename(chainId)
+  const assetCollDeployments = <IAssetCollDeployments>getDeploymentFile(assetCollDeploymentFilename)
+
+  const deployedCollateral: string[] = []
+
+  /********  Deploy MetaMorpho Morpho eUSD - meUSD  **************************/
+
+  // Only for base
+  if (baseL2Chains.includes(hre.network.name)) {
+    const MetaMorphoFiatCollateralFactory: ContractFactory = await hre.ethers.getContractFactory(
+      'MetaMorphoFiatCollateral'
+    )
+
+    const collateral = <MetaMorphoFiatCollateral>await MetaMorphoFiatCollateralFactory.connect(
+      deployer
+    ).deploy(
+      {
+        priceTimeout: PRICE_TIMEOUT.toString(),
+        chainlinkFeed: eUSD_USD_FEED,
+        oracleError: eUSD_ORACLE_ERROR.toString(),
+        erc20: networkConfig[chainId].tokens.meUSD,
+        maxTradeVolume: fp('1e6').toString(), // 17m vault
+        oracleTimeout: eUSD_ORACLE_TIMEOUT.toString(),
+        targetName: hre.ethers.utils.formatBytes32String('USD'),
+        defaultThreshold: eUSD_ORACLE_ERROR.add(fp('0.01')).toString(), // +1% buffer rule
```

Comment:
do we need to add a full 1% here? we're targetting base, so the oracles should be quick to update, and the base oracle error is 0.5%. feels like we could get away with 1% total? (oracle error + 0.5%). @tbrent thoughts?

Reply:
The +1% rule doesn't interact with the attributes of the oracle in my mind, eg oracleError or oracleTimeout, etc. 

Anyway, the oracleTimeout is 24h. I don't see why it being on base would change anything, this is still the value that governs how often they will push heartbeat updates.